src/components/validators/project-name-available-validator.js

```diff
@@ -6,6 +6,9 @@
       return {
         restrict: 'A',
         require: 'ngModel',
+        scope: {
+          organizationId: "="
```

Comment:
Yes, just there's different taste between different projects - so better keeping some consistency, as much as possible for new implementation.